fms/modules/head.py

```diff
@@ -75,3 +84,89 @@ def forward(self, x: torch.Tensor):
             x = self.ln(x)
         x = self.head(x)
         return x
+
+
+class LMHead(nn.Linear):
```

Comment:
maybe LMLinearHead? I'm assuming that after separating them there might be multiple kinds of heads.

Is there a particular use-case motivating this (and TPLMHead) instead of creating a TPClassificationHead based on the head defined above?

Reply:
I'll just use ClassificationHead. For some reason I had it in my head that an LMHead wasn't just a ClassificationHead. I'll rework this PR and the stack in general to use this.

Reply:
Actually nevermind, the ClassificationHead we have is a little too complicated: it has two Linear layers, a non-linearity, dropout, etc., while most LM Heads are only a single linear layer (+ bias sometimes)

Reply:
hmmm....  What about naming them something more specific? i.e. maybe we could make these something like MLPClassificationHead and LinearClassificationHead so they're better differentiated from each other (and from whatever other kinds of heads we might add later)

Reply:
That makes sense, I'll make the changes